### ISSUE TITLE
DefaultJsonMapper throws error when converting Post jsonString to an object of Post class.

### ISSUE THREAD

**Issue by MahorShekhar** [ebx/ebx-linkedin-sdk#298]
Hi, I'm converting a Post class jsonString to its object.

```
        String expectedPostJson = "{\"author\":\"urn:li:organization:social_network_id\",\"commentary\":\"sample post text\",\"content\":{\"multiImage\":{\"images\":[{\"id\":\"urn:li:image:imageId1\"},{\"id\":\"urn:li:image:imageId2\"}]}},\"distribution\":{\"feedDistribution\":\"MAIN_FEED\"},\"lifecycleState\":\"PUBLISHED\",\"visibility\":\"PUBLIC\"}";

        JsonMapper jsonMapper = new DefaultJsonMapper();
        Post p = jsonMapper.toJavaObject(expectedPostJson, Post.class);
```

But I'm getting exception

```
com.echobox.api.linkedin.exception.LinkedInJsonMappingException: Unable to create an instance of class com.echobox.api.linkedin.types.posts.MultiImageContent. Please make sure that if it's a nested class, is marked 'static'. It should have a no-argument constructor.

	at com.echobox.api.linkedin.util.ReflectionUtils.createInstance(ReflectionUtils.java:411)
	at com.echobox.api.linkedin.jsonmapper.DefaultJsonMapper.toJavaObject(DefaultJsonMapper.java:279)
	at com.echobox.api.linkedin.jsonmapper.DefaultJsonMapper.toJavaType(DefaultJsonMapper.java:747)
	at com.echobox.api.linkedin.jsonmapper.DefaultJsonMapper.toJavaObject(DefaultJsonMapper.java:320)
	at com.echobox.api.linkedin.jsonmapper.DefaultJsonMapper.toJavaType(DefaultJsonMapper.java:747)
	at com.echobox.api.linkedin.jsonmapper.DefaultJsonMapper.toJavaObject(DefaultJsonMapper.java:320)
	at com.bcv.pipeline.process.linkedin.content.LinkedInContentProcessTest.testSuccess(LinkedInContentProcessTest.java:29)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: java.lang.NoSuchMethodException: com.echobox.api.linkedin.types.posts.MultiImageContent.<init>()
	at java.base/java.lang.Class.getConstructor0(Class.java:3585)
	at java.base/java.lang.Class.getDeclaredConstructor(Class.java:2754)
	at com.echobox.api.linkedin.util.ReflectionUtils.createInstance(ReflectionUtils.java:400)
	... 33 more

```

Can somebody please help me resolving this issue. 🙏

**Comment by MahorShekhar** [comment]
@diepjy @kennethwong-hc please assist with this.

**Comment by djjashton** [comment]
Hi

Thank you for raising this issue. I have tested this out and found that it was fix in the 4.1.2 release. Please see https://github.com/ebx/ebx-linkedin-sdk/pull/306.
